electricitylci/eia_io_trading.py

```diff
@@ -133,6 +133,7 @@ def ba_io_trading_model(year=None, subregion=None, regions_to_keep=None):
 #                TOTAL_INTERCHANGE_ROWS.append(json.loads(line))
     logging.info(f"Net gen rows: {len(NET_GEN_ROWS)}; BA to BA rows:{len(BA_TO_BA_ROWS)}; Demand rows:{len(DEMAND_ROWS)}")
     eia923_gen=eia923.build_generation_data(generation_years=[year])
+    eia923_gen=eia923_gen.drop_duplicates()
```

Comment:
@ssamkl, to double-check that this is necessary.

Reply:
@m-jamieson, Hey Matt, why did you add these drop_duplicate functions? Did you find duplicates on your end when running the code? Because when we ran the code, there were no duplicates found. This comment applies to drop_duplicate functions used in eia_trans_dist_grid_loss.py and generation_mix.py.

Reply:
That change was made weeks ago - I can't remember! I think this was probably just my attempts to troubleshoot without actually debugging. I believe the problem was eventually resolved with the change to the plant_water_use.py in the same commit. I'd say this can probably go away. At the same time - what harm could they possibly do?

Reply:
Understood! And apologies for the delay. Thank you for letting us know :)